container/features/src/main/resources/features.xml

```diff
@@ -1111,6 +1111,13 @@
       <bundle>mvn:org.opennms.features.poller/org.opennms.features.poller.shell/${project.version}</bundle>
     </feature>
 
+    <feature name="opennms-situation-feedback" description="OpenNMS :: Features :: Situation-Feedback" version="${project.version}">
```

Comment:
It looks like there are missing dependencies here - the feature should install cleanly in an empty container.

We may want to add a test case in https://github.com/OpenNMS/opennms/blob/develop/opennms-full-assembly/src/test/java/org/opennms/assemblies/karaf/OnmsFeatureKarafIT.java for it.